server/app/repository/ExportServiceRepository.java

```diff
@@ -1,116 +1,59 @@
 package repository;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import com.google.common.collect.ImmutableList;
-import com.google.inject.Provider;
 import io.ebean.DB;
 import io.ebean.Database;
 import java.util.LinkedHashSet;
-import java.util.List;
 import javax.inject.Inject;
-import models.Application;
-import models.LifecycleStage;
-import models.Version;
-import services.applicant.AnswerData;
-import services.applicant.ApplicantService;
-import services.applicant.ReadOnlyApplicantProgramService;
-import services.question.types.MultiOptionQuestionDefinition;
 import services.question.types.QuestionDefinition;
 import services.question.types.QuestionType;
 
 /** Implements queries related to CSV exporting needs. */
 public final class ExportServiceRepository {
   private final Database database;
-  private final Provider<VersionRepository> versionRepositoryProvider;
-  private final ApplicantService applicantService;
 
   @Inject
-  public ExportServiceRepository(
-      Provider<VersionRepository> versionRepositoryProvider, ApplicantService applicantService) {
+  public ExportServiceRepository() {
     this.database = DB.getDefault();
-    this.versionRepositoryProvider = checkNotNull(versionRepositoryProvider);
-    this.applicantService = checkNotNull(applicantService);
   }
 
   /**
-   * This method queries for all submitted applications in the database ordered by their createTime
-   * and checks if they have used the question passed in the input param. If the question is
-   * present, it creates a unique set of all options ever picked along with the question's active
-   * version options and creates the return list.
+   * This method queries for all unique option admin names ordered by is creation time.
```

Comment:
```suggestion
   * Queries for all unique option admin names ordered by their creation time.
```

https://google.github.io/styleguide/tsguide.html#method-and-function-comments
>Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.